helm/app/templates/dashboard-role-binding.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: RoleBinding
+metadata:
+  name: {{ include "app.fullname" $ }}
+  {{- if hasKey $values "namespace" }}
+  namespace: {{ $values.namespace }}
+  {{- end }}
+  labels:
+    helm.sh/chart: {{ include "app.chart" $ }}
+    app.kubernetes.io/name: {{ include "app.name" $ }}
+    app.kubernetes.io/instance: {{ $.Release.Name }}
+    app.kubernetes.io/managed-by: {{ $.Release.Service }}
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: go-api-boilerplate-dashboard
+subjects:
+- kind: ServiceAccount
+  name: dash-kubernetes-dashboard
+  namespace: default
```

Comment:
missing new line at the end of file

Reply:
How do you detect those when the build reports success ?